src/ActionUnit.cpp

```diff
@@ -443,7 +458,21 @@ void ActionUnit::constructToolbar(TAction* pA, TToolBar* pTB)
     }
 
     pTB->clear();
-    if ((pA->mLocation != 4) || (!pA->isActive())) {
```

Comment:
I am not :100: sure I have the logic correct here - if `pA->mLocation` is NOT `4` then it definitely NOT a (dockable/floating) `TToolBar` - but it need not be a fixed around the edge of the profile's main console `TEasyButtonBar` either. OTOH Maybe I need the following here AS WELL as separately done for the `!pA->isAactive()` case:
```cpp
        pTB->setFloating(false);
        mudlet::self()->removeDockWidget(pTB);
        return;
```
?

Reply:
Well, everything seems to work so perhaps I have gotten it sorted.

:bulb: In the medium term I do want to revisit the buttons/menus/toolbars code so that elements that are disable are not automagically hidden from view - I'd be proposing to add a checkbox option (defaulting to checked to maintain previous behaviour) "hide when deactivated" - which would need to slot into the code in various places including this one probably...